pavelib/servers.py

```diff
@@ -213,8 +213,8 @@ def run_all_servers(options):
         # means that the optimized assets are ignored, so we skip collectstatic in that
         # case to save time.
         if settings != DEFAULT_SETTINGS:
-            collect_assets(['lms'], asset_settings_lms)
-            collect_assets(['studio'], asset_settings_cms)
+            collect_assets(['lms'], asset_settings_lms, collectstatic_log="/dev/null")
+            collect_assets(['studio'], asset_settings_cms, collectstatic_log="/dev/null")
```

Comment:
This just uses the default behavior right now, right? so there's no need to explicitly specify `/dev/null`.